be/src/util/simd/vstring_function.h

```diff
@@ -122,6 +122,101 @@ class VStringFunctions {
         return rtrim(ltrim(str));
     }
 
+    static StringRef rtrim(const StringRef& str, const StringRef& rhs) {
+        if (str.size == 0 || rhs.size == 0) {
+            return str;
+        }
+        if (rhs.size == 1) {
```

Comment:
not required for this pr but may be considerable: for most situations these function to be used, the needles are likely to be constant. So we can also use SIMD bitwise compare to optimize them. just give code below a little change will make it.